Libraries/ActionSheetIOS/NativeActionSheetManager.js

```diff
@@ -47,6 +47,7 @@ export interface Spec extends TurboModule {
     |}) => void,
     successCallback: (completed: boolean, activityType: ?string) => void,
   ) => void;
+  +dismissActionSheet: () => void;
```

Comment:
Can we make this a nullable function? This will break codebases that leverage OTA type safety in the case where this JS is running on a native app where this property isn't defined.

Reply:
Sure, I've just updated it